ExampleGame/Assets/TestUI/TestUI.cs

```diff
@@ -27,7 +27,7 @@ public class TestUI : MonoBehaviour{
 		if(GUI.Button(GetControlRect(1), "Divide By Zero"))
 		{
 
-			int i = 0;
+			decimal i = 0;
```

Comment:
Add the comment with explanation and link to documentation that prove it

Reply:
Yes, please